.github/workflows/push-to-deploy.yml

```diff
@@ -0,0 +1,20 @@
+name: Deploy to WordPress.org
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  tag:
+    name: New tag
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Build
+      run: |
+        npm install
+        npm run build
+    - name: WordPress Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@master
```

Comment:
use action `Nikschavan/action-wordpress-plugin-deploy@develop` for now until there is a resolution to https://github.com/10up/action-wordpress-plugin-deploy/issues/1

Reply:
Done with the changes.